libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -84,6 +84,7 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     float iblLuminance;
     float exposure;
     float ev100;
+    float needsAlphaChannel;
 
```

Comment:
Uniforms are arranged in groups of `float4`. You'll need to add padding, otherwise the alignment of other fields slips.
```suggestion
    float ev100;
 
    float needsAlphaChannel;
    math::float3 padding3; 
```